extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -148,6 +160,45 @@ export class IframeTransport {
   }
 
   /**
+   * Uses the Long Task API to create an observer for when 3p vendor frames
+   * take more than 50ms of continuous CPU time.
+   * Currently the only action in response to that is to log. There is a
+   * grace of LONG_TASK_REPORTING_THRESHOLD_ occurrences before logging begins.
+   * @VisibleForTesting
+   */
+  createLongTaskObserver() {
+    if (this.longTaskObserver_ || !this.ampWin_.PerformanceObserver) {
```

Comment:
I feel you could use this `isLongTaskApiSupported` function from jank-meter.js

Reply:
Done